.github/pull_request_template.md

```diff
@@ -21,3 +21,7 @@ List impacted areas.
 - [ ] I have commented my code, particularly in hard-to-understand areas
 - [ ] I have made corresponding changes to the documentation
 - [ ] My changes generate no new warnings
+- [ ] I have cleaned up integration tests related to my changed by doing one of the following:
```

Comment:
one or more of the following

last one: I would more directly call out the conversion of use-fixtures :each to :once

Reply:
updated!